deb/openmediavault/workbench/src/app/pages/diagnostics/system-logs/system-logs-remote-form-page.component.ts

```diff
@@ -89,6 +89,9 @@ export class SystemLogsRemoteFormPageComponent extends BaseFormPageComponent {
           url: '/diagnostics/system-logs'
         }
       }
-    ]
+    ],
+    styling: {
+      cardClasses: 'omv-w-xs'
```

Comment:
Not the best solution to have the need to touch every form page to specify the styling.

Reply:
the other way 'round would be to make every form smaller by default, see https://github.com/openmediavault/openmediavault/pull/1575#discussion_r1290175835